aws.yaml

```diff
@@ -1,4 +1,4 @@
-- active: false
+- active: true
```

Comment:
aws-operator@5.5.0 is a legacy branch so we'd need to release a flat operator. Please ping @giantswarm/team-magic  when doing so for a while.

@giantswarm/team-magic needs to fix one bug with finalizers before we can deploy that to production. It was discovered yesterday evening. I'll link to the issue once it's created.

Reply:
Here's the issue https://github.com/giantswarm/giantswarm/issues/7376

Reply:
Fix is merged and vendored now..